test/ui/transfer-item.test.ts

```diff
@@ -58,9 +58,9 @@ describe("UI - Transfer Items", () => {
 
   it("check red tint for held item limit in transfer menu", async () => {
     game.onNextPrompt("SelectModifierPhase", UiMode.PARTY, () => {
-      expect(game.scene.ui.getHandler()).toBeInstanceOf(PartyUiHandler);
+      expect(game.scene.ui.getCurrentHandler()).toBeInstanceOf(PartyUiHandler);
 
-      const handler = game.scene.ui.getHandler() as PartyUiHandler;
+      const handler = game.scene.ui.getCurrentHandler() as PartyUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
      const handler = game.scene.ui.getCurrentHandler<PartyUiHandler>();
```